x-pack/plugins/index_management/public/application/sections/home/data_stream_list/data_stream_detail_panel/data_stream_detail_panel.tsx

```diff
@@ -327,6 +328,33 @@ export const DataStreamDetailPanel: React.FunctionComponent<Props> = ({
       },
     ];
 
+    // If both rentention types are available, we wanna surface to the user both
+    if (lifecycle?.effective_retention && lifecycle?.data_retention) {
+      defaultDetails.push({
+        name: i18n.translate(
+          'xpack.idxMgmt.dataStreamDetailPanel.customerDefinedDataRetentionTitle',
+          {
+            defaultMessage: 'Data retention',
+          }
+        ),
+        toolTip: i18n.translate(
+          'xpack.idxMgmt.dataStreamDetailPanel.customerDefinedDataRetentionTooltip',
+          {
+            defaultMessage: `This is the data retention defined by you, but the effective retention is the one being enforced.`,
```

Comment:
```suggestion
            defaultMessage: "This is the data retention that you defined. Because of other system constraints or settings, the data retention that is effectively applied may be different from the value you set. You can find the value retained and applied by the system under 'Effective data retention'.",
```
Not entirely sure how clear my suggestion is. I want to use this tooltip to give some sort of explanation so that users don't get too confused as to why the effective data retention is different from what they're trying to set.